docs/operational_guide/monitoring.md

```diff
@@ -0,0 +1,74 @@
+## Metrics
+
+TODO
+
+## Logs
+
+TODO
+
+## Tracing
+
+M3DB is integrated with [opentracing](https://opentracing.io/) to provide
+insight into query performance and errors.
+
+### Configuration
+Currently, only https://www.jaegertracing.io/ is supported as a backend.
```

Comment:
nit: maybe `only [Jaeger](https://www.jaegertracing.io/) is supported as a tracing backend.`